integration-test/1425-more-osm-features.py

```diff
@@ -926,6 +926,29 @@ def test_plaque_node(self):
                 'kind': u'plaque',
             })
 
+    def test_memorial_node(self):
+        # if a memorial _isn't_ a plaque, then it should stay as a memorial!
```

Comment:
Worth a note in the changelog!

Reply:
Added https://github.com/tilezen/vector-datasource/issues/1872 to remind us to put into changelog.